os/net/lwip/src/api/sockets.c

```diff
@@ -84,6 +84,7 @@
 #include <net/lwip/tcpip.h>
 #include <net/lwip/pbuf.h>
 #include <net/lwip/mem.h>
+#include <arpa/inet.h>
```

Comment:
Why include the header file here, if the rest of the file does not use it? Isn't it better to include files where they are used directly?